package estudomicroservice.avaliadorcredito.application.exception;

public class DadosClientNoutFoundException extends  Exception {
    public DadosClientNoutFoundException() {
        super("Dados do cliente n√£o encontrados para o CPF informado!");
    }
}
